### ISSUE TITLE
food for thought

### ISSUE THREAD

**Issue by MK-HATERS** [ejams1/starfield-modding-guide#6]
not an issue but i used the [starfield plugin bridge](https://www.nexusmods.com/starfield/mods/10189), and bruh removing blueprintships from most of the mods 30/45 it let me i saw a huge performance gain, figured id say try it out and see about adding it to your list

**Comment by ejams1** [comment]
Oh nice! I had read about that, but couldn't understand what the process was to _just_ remove the blueprint dependency. Can you summarize it here, and I will add it to the guide? I'm looking for all the performance gains possible with this game hah

**Comment by MK-HATERS** [comment]
i would also suggest root builder,
but basically you do a loot run get which mods have bps.esm as master
open up those installed mods and drag onot the plugin bridge exe, it will create a .stubbed.esm
once you have stbbed you run that and the esm mod that comes with bridge, and you xedit them and clean master in xedit
xedit will give errors on mids and some other ones itll give you a warning, basically it has a safety net that wont change ones it shouldnt
now replace fixed stubbed esm to replace the old one or just move it to optional file path so mo2 ignores it

you are better explaining and i think you need a hands on with it either way it takes time but worth it

**Comment by ejams1** [comment]
Appreciate it! I will investigate it tonight, thanks.

I'm also a fan of root builder, but the reason I used stock game folder is to preserve the function of my list even if Bethesda updates Starfield which is highly likely given the game is only a year old and just had its DLC release recently. With root builder, an update from Bethesda will break everything but stock game folder keeps your older executable safe with that version of the game.

**Comment by ejams1** [comment]
I ended up following the alternate advice from a post in the comments about removing the flag in the CK which was much quicker - just had to put all the ESMs I wanted to remove the flag for into the `base game/Data` folder, then launch the CK and open Data and remove the master from each.

I'm cleaning up my load order but will report back once I get in-game to test, for now LOOT is reporting no issues so they have all been cleaned properly.

**Comment by ejams1** [comment]
I forgot you mentioned xEdit won't remove the flag from mods that it shouldn't which the CK method does not do so I guess I will see what happens 😄

**Comment by ejams1** [comment]
I'm not seeing any issues, and I'm also not seeing any significant performance gain even though I had ~25 plugins with that as master. If you are seeing improvements, I think the likely explanation is that something in your load order was loading improperly due to the master override.

Regardless, I appreciate the call-out and this addresses a big issue with improper load order which I have now added to the guide so thank you!

**Comment by MK-HATERS** [comment]
no worries this is what open-source is about! we can only get better studying, this is the way
